android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/InstallDialogActivity.java

```diff
@@ -54,7 +54,7 @@
   private static String installButtonTextColor = "#ffffffff";
   private final String GOOGLE_PLAY_URL =
       "https://play.google.com/store/apps/details?id=" + BuildConfig.BDS_WALLET_PACKAGE_NAME;
-  private final String CAFE_BAZAAR_URL = "https://cafebazaar.ir/app/com.hezardastaan.wallet";
+  private final String CAFE_BAZAAR_URL = "bazaar://details?id=com.hezardastan.wallet";
```

Comment:
2 different urls ?

Reply:
There was, this PR is to put the same one in both flows